tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3786,12 +3770,13 @@ node_modules/chrome-devtools-frontend/front_end/common/Throttler.js(97,14): erro
 node_modules/chrome-devtools-frontend/front_end/common/Throttler.js(102,5): error TS2322: Type 'Timer' is not assignable to type 'number'.
 node_modules/chrome-devtools-frontend/front_end/common/Throttler.js(113,15): error TS7014: Function type, which lacks return-type annotation, implicitly has an 'any' return type.
 node_modules/chrome-devtools-frontend/front_end/common/Throttler.js(114,18): error TS2339: Property 'FinishCallback' does not exist on type 'typeof Throttler'.
+node_modules/chrome-devtools-frontend/front_end/common/UIString.js(32,1): error TS2322: Type 'typeof Common | {}' is not assignable to type 'typeof Common'.
```

Comment:
`self["Common"] = self["Common"] || {}` no longer works because we can now resolve `Common` as a global, where before it was `any`.